openshift-ci/Dockerfile.registry.intermediate

```diff
@@ -0,0 +1 @@
+FROM quay.io/openshift/origin-operator-registry:latest
```

Comment:
What's the use of this dockerfile?

Reply:
NVM, I see it being used in openshift/release#4931. And I've gotta question on it there.

Reply:
yep, I responded there. I'll give a brief explanation here as well.

I included separate Dockerfiles for the Openshift CI job because we're having to deploy some custom logic around getting the CSV in ocs-registry to point to the ocs-operator container image built by the CI job, and not use quay.io/ocs-dev/ocs-operator:latest.